src/javascript/binary/pages/portfiliows.js

```diff
@@ -0,0 +1,181 @@
+var PortfolioWS =  (function() {
+
+    'use strict';
+
+    var pageLanguage = page.language();
+    if(!pageLanguage) pageLanguage = "EN";
+
+    var rowTemplate;
+    var retry;
+
+    var init = function() {
+        // get the row template and then discard the node as it has served its purpose
+        rowTemplate = $("#portfolio-dynamic tr:first")[0].outerHTML;
+        $("#portfolio-dynamic tr:first").remove();
+        TradeSocket.init({"thendo":"updateBalance"});
```

Comment:
Not sure if it violates SRP, there might be valid reasons to return data that is available via other means. In this case I think it is rather random ;)
I too am using this value to initialize the balance value within binary-next-gen, but will redo it with balance streaming and guess no need to bother with this entry (and we should just take it out)